### ISSUE TITLE
digitalRead

### ISSUE THREAD

**Issue by fn-code** [rwaldron/johnny-five#1425]
how to stop read data from digitalRead.???

thank you..

**Comment by dtex** [comment]
@fn-code Sorry nobody has responded yet. Are you talking about the digitalRead in firmata.js? I found this in the firmata.js readme:

> To stop reporting digital values for a pin, call board.reportDigitalPin(digitalPinNumber, 0)

In Johnny-Five that probably should be ```board.io.reportDigitalPin(digitalPinNumber, 0)```

**Comment by dtex** [comment]
HI @fn-code just following up. Did this help?

**Comment by fn-code** [comment]
Yeah, its work thank you. But, when i started another digitalRead in the same pin. The old digitalRead start read data again??
